### DISCUSSION TITLE
Questions about ann-to-snn conversion

### DISCUSSION THREAD

**Discussion by MayWeHappy** [BindsNET/bindsnet#628] [Q&A]
Dear Authors,

I trained an MLP on fashionMNIST with 2 hidden layers and converted it to SNN using your ann_to_snn function, with database normalization. However, testing shows the SNN generates the same output for all inputs. 

I believe I trained the MLP properly but parameter selection may be wrong for the SNN. Adjusting SNN parameters like thresholds/weight decay could resolve this, allowing effective feature extraction and classification. 

How can I modify parameters to fix this? Any guidance would be appreciated.

GuYe

**Comment by Hananel-Hazan** [comment]
Its recommended to run a heuristic search on the parameters to find the best match.
